index.js

```diff
@@ -148,8 +149,22 @@ async function handleAlert(location, alertType) {
         fallbackMessage: `An alert to check on the bathroom at ${location.displayName} was not responded to. Please check on it`,
         fallbackToPhoneNumbers: location.fallbackNumbers,
         fallbackFromPhoneNumber: location.twilioNumber,
+        sirenParticleId: location.siren_particle_id,
       }
-      braveAlerter.startAlertSession(alertInfo)
+
+      if (location.siren_particle_id != null) {
+        try{
+          await axios.post('https://api.particle.io/v1/devices/location.siren_particle_id/start-siren', {
+            arg: 'start',
+            access_token: process.env.ACCESS_TOKEN
```

Comment:
I suggest calling the token `PARTICLE_ACCESS_TOKEN` just because it's possible that we end up with a few third-party tools that also call their things access tokens, lol.

What you've done here is the correct way to get at an environment variable; however, we've come up with our own helper function to do it for us because it gives a different value when we run the tests than when we are in a real dev, staging, or production environment. So instead of `process.env.ENV_VAR_NAME` we use `helpers.getEnvVar('ENV_VAR_NAME')`.